var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -165,6 +165,11 @@ class Sirius(CMakePackage, CudaPackage, ROCmPackage):
         conflicts("^spla@:1.5~openmp", when="+openmp")
 
     patch("libxc7.patch", when="@7.6:")
+    patch(
+        "https://patch-diff.githubusercontent.com/raw/electronic-structure/SIRIUS/pull/1050.patch?full_index=1",
```

Comment:
can you use the commit url instead of the PR url? https://github.com/electronic-structure/SIRIUS/commit/dd07010f7b49f31b7e3bb1b4e47f3d9ac3a0c0b4. Idk if the former stays forever if branches are deleted or accidentally pushed to.

Reply:
Good point, thanks. Done.